### ISSUE TITLE
[Orphaned Module]: `avm-ptn-dev-center-dev-box`

### ISSUE THREAD

**Issue by autocloudarc** [Azure/Azure-Verified-Modules#1527]
### Bicep or Terraform?

Terraform

### Module Classification?

Pattern Module

### Module Name

Terraform: avm-ptn-dev-center-dev-box

### Module Details

Requesting ownership. See also issue: #759 

### Do you want to be the new owner of this module?

Yes

### Module Owner's GitHub Username (handle)

autocloudarc

**Comment by prjelesi** [comment]
@autocloudarc wince this is request for pattern module, can you please provide more info about what module will deploy and if there is any architecture design for this?

**Comment by autocloudarc** [comment]
Yes, this will deploy an Azure DevCenter then configure the devbox definition image, devbox pool, devbox task catalog to support the deployment of a devbox from the devbox pool via the developer portal at: https://devportal.microsoft.com/.

**Comment by autocloudarc** [comment]
Here's a better diagram of the devbox architecture. 

![Image](https://github.com/user-attachments/assets/bf7d8432-9e92-4209-b810-93c5399e0a7e)
